### PR TITLE
:ambulance: Orders history is not updated instantly

### PR DESCRIPTION
**Pull request by GabbasovDinar**

### CODE REVIEW

**Review by ilmir-k — APPROVED** [review]

File: pos_orders_history/static/src/js/screens.js [hunk 1]
```diff
@@ -151,6 +151,10 @@ odoo.define('pos_orders_history.screens', function (require) {
                 });
             }
         },
+        get_datetime_format: function(datetime) {
+            var d = new Date(datetime);
+            return new Date(Date.UTC(d.getFullYear(), d.getMonth(), d.getDate(), d.getHours(), d.getMinutes(), d.getSeconds())).toLocaleString();
```
Comment (yelizariev) [inline pos_orders_history/static/src/js/screens.js]: What the hell? Can you simply use ``d.toLocaleString();``

**Review by yelizariev — CHANGES_REQUESTED** [review]

File: pos_orders_history/static/src/js/screens.js (hunk 1 above)
Comment (GabbasovDinar) [reply pos_orders_history/static/src/js/screens.js]: image

Comment (yelizariev) [reply pos_orders_history/static/src/js/screens.js]: Do you agree or not?

Comment (GabbasovDinar) [reply pos_orders_history/static/src/js/screens.js]: @yelizariev No

Comment (yelizariev) [reply pos_orders_history/static/src/js/screens.js]: `utc =  new Date(d.getTime() - d.getTimezoneOffset() * 60000)`

**Comment by GabbasovDinar** [comment]
@yelizariev DONE

**Comment by ilmir-k** [comment]
@yelizariev can we proceed?

**Review by yelizariev — APPROVED** [review]

**Pull request merged**